sysdeps/linux/generic/sysdeps.cpp

```diff
@@ -449,7 +449,7 @@ void sys_exit(int status) {
 #define FUTEX_WAIT 0
 #define FUTEX_WAKE 1
 
-int sys_futex_wait(int *pointer, int expected) {
+int sys_futex_wait(int *pointer, int expected, int seconds, int nanoseconds) {
 	auto ret = do_cp_syscall(NR_sys_futex, pointer, FUTEX_WAIT, expected, nullptr);
```

Comment:
The syscall can receive a ``struct timespec *``, so I'd suggest adding that here for completeness sake. (and so that we don't get weird bugs when testing timeouts on linux in the future)

Reply:
Would that be the `nullptr` argument on line 453?

Reply:
Yes

Reply:
Okay in the new code why can't we just pass the pointer directly?